lib/types/mod.ex

```diff
@@ -43,4 +43,8 @@ defmodule Types do
   @type execution_address :: bytes20
   @type withdrawal_index :: uint64
   @type payload_id :: bytes8
+  @type kzg_commitment :: bytes48
+  @type kzg_proof :: bytes48
+  ## 131072 is FIELD_ELEMENTS_PER_BLOB(i.e 4096) * BYTES_PER_FIELD_ELEMENT(i.e 32)
+  @type blob :: <<_::131_072>>
```

Comment:
This varies between chains, so we should leave it as `binary()`. Also, let's move all three to `Kzg` and try marking them as `@opaque`.

Reply:
Can you please reference where this is stated(that the size of a blob varies across chains)?

Reply:
Since it's part of the preset, it can change between chains. However both minimal and mainnet use the same number, so we can leave it hardcoded for now.